common/djangoapps/course_modes/models.py

```diff
@@ -26,13 +27,17 @@
                   ])
 
 
+class classproperty(property):
```

Comment:
What is the difference between this and https://docs.python.org/3/library/functions.html#staticmethod?